src/Microsoft.Performance.SDK.Runtime/CustomDataSourceReference.cs

```diff
@@ -61,36 +113,72 @@ protected CustomDataSourceReference(Type type, Func<ICustomDataSource> instanceF
         /// <exception cref="System.ObjectDisposedException">
         ///     <paramref name="other"/> is disposed.
         /// </exception>
-        protected CustomDataSourceReference(CustomDataSourceReference other)
+        public CustomDataSourceReference(CustomDataSourceReference other)
```

Comment:
do we want to copy createdprocessor[ro]?

Reply:
Yes, I will look at it.